extensions/azdata/src/test/common/childProcess.test.ts

```diff
@@ -9,21 +9,22 @@ import * as TypeMoq from 'typemoq';
 import { executeCommand } from '../../common/childProcess';
 
 describe('ChildProcess', function () {
-	[undefined, [], ['test']].forEach(args => {
+	const outputChannelMock = TypeMoq.Mock.ofType<vscode.OutputChannel>();
+
+	[[], ['test']].forEach(args => {
 		it(`Output channel is used with ${JSON.stringify(args)} args`, async function (): Promise<void> {
-			const outputChannelMock = TypeMoq.Mock.ofType<vscode.OutputChannel>();
 			await executeCommand('echo', args, outputChannelMock.object);
-			outputChannelMock.verify(x => x.appendLine(TypeMoq.It.isAny()), TypeMoq.Times.once());
+			outputChannelMock.verify(x => x.appendLine(TypeMoq.It.isAny()), TypeMoq.Times.atLeastOnce());
```

Comment:
Why not verify the actual contents of the call to appendLine? you know that should contain 'echo'

Reply:
Because that's getting to a level of detail I don't find valuable in tests. Honestly I usually don't even care whether the output channel was called - that's more of an implementation/side detail rather than actual functionality I care about testing.